nur/prefetch.py

```diff
@@ -13,9 +13,13 @@
 
 
 def fetch_commit_from_feed(url: str) -> str:
-    req = urllib.request.urlopen(url)
+    req = urllib.request.Request(
+        url,
+        headers={"User-Agent": "NixUserRepositoryUpdater"}
```

Comment:
Dunno if this is something that's nice to have for all requests or just GitLab. I'm thinking it might be nice to have in general so that its requests are somewhat traceable back to NUR.

Also I didn't spend much effort on naming so suggestions are appreciated 🙂